packages/devtools_app/lib/src/screens/memory/shared/primitives/class_name.dart

```diff
@@ -2,20 +2,65 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
 import 'package:vm_service/vm_service.dart';
 
 import '../../../../shared/primitives/simple_items.dart';
+import '../../../../shared/ui/icons.dart';
+
+enum ClassType {
+  runtime(
+    color: Color.fromARGB(255, 238, 109, 99),
+    label: 'R',
+    alias: '\$runtime',
+    aliasDescription: 'Dart runtime classes',
+    classTooltip: 'Dart runtime class',
+  ),
+  sdk(
+    color: Color.fromARGB(255, 122, 188, 124),
```

Comment:
Are these standard colors? Can they be made into constants and placed with our other colors?

Reply:
I plan to do it outside of this PR.

@malloc-error plans to work on color scheme for DevTools
There is issue to handle this: https://github.com/flutter/devtools/issues/5050